gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/test/features/map/Project.feature

```diff
@@ -106,4 +106,13 @@ Feature: Step - project()
       | m[{"a":"d[0].l", "b":"d[29].i"}] |
       | m[{"a":"d[1].l", "b":"d[27].i"}] |
       | m[{"a":"d[1].l", "b":"d[32].i"}] |
-      | m[{"a":"d[0].l", "b":"d[35].i"}] |
\ No newline at end of file
+      | m[{"a":"d[0].l", "b":"d[35].i"}] |
+
+
+  Scenario: g_V_projectXa_aX
```

Comment:
Looks like the Gherkin tests can't handle cases that fast fail on step construction (I believe it needs the step to be successfully constructed before translating it into the GLV tests). I don't think there is any easy work around currently, sorry @rdtr, you might need to remove this test and just have unit tests.

(As an aside, the test is missing a `When iterated to list`, but that's not really important now that this test won't really work in Gherkin).

Reply:
Actually there is a quick workaround I just remembered. You can technically add static translation of this test to all the feature tests in the `generate.groovy` file of each GLV, but it's ultimately a hack so I'd think it's ok to not include it at this point.

Reply:
Sorry I missed that this scenario is incompatible with the feature tests.

VOTE +1 (with failing feature test removed)

Reply:
Ok, I deleted the feature test. Please check when you have time, thanks !